topics/statistics/tutorials/CNN/data-library.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Please ignore the file contents here. I will create a new data library after workflow can be run.

Reply:
Contents are good now.